checkov/terraform/checks/resource/aws/ECSClusterLoggingEnabled.py

```diff
@@ -0,0 +1,26 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+from typing import Any, List
+
+
+class ECSClusterLoggingEnabled(BaseResourceNegativeValueCheck):
+    def __init__(self):
+        name = "Ensure ECS Cluster enables logging"
```

Comment:
```suggestion
        name = "Ensure ECS Cluster enables logging of ECS Exec"
```
I would add an extra info, that these are "just" the ECS Exec logs, not cluster logs